### PR TITLE
 Added horrible python one-liner solution to day03 (And day01-03 solutions in golang!!!)

### PR DESCRIPTION
**Pull request by Stektpotet**
## for 
# extra 
```
 ▄▄   ▄▄ ▄▄    ▄ ▄▄▄▄▄▄   ▄▄▄▄▄▄▄ ▄▄▄▄▄▄▄ ▄▄▄▄▄▄  ▄▄▄▄▄▄▄ ▄▄▄▄▄▄▄ ▄▄▄ ▄▄▄     ▄▄▄ ▄▄▄▄▄▄▄ ▄▄   ▄▄ 
█  █ █  █  █  █ █   ▄  █ █       █       █      ██       █  ▄    █   █   █   █   █       █  █ █  █
█  █ █  █   █▄█ █  █ █ █ █    ▄▄▄█   ▄   █  ▄    █   ▄   █ █▄█   █   █   █   █   █▄     ▄█  █▄█  █
█  █▄█  █       █   █▄▄█▄█   █▄▄▄█  █▄█  █ █ █   █  █▄█  █       █   █   █   █   █ █   █ █       █
█       █  ▄    █    ▄▄  █    ▄▄▄█       █ █▄█   █       █  ▄   ██   █   █▄▄▄█   █ █   █ █▄     ▄█
█       █ █ █   █   █  █ █   █▄▄▄█   ▄   █       █   ▄   █ █▄█   █   █       █   █ █   █   █   █  
█▄▄▄▄▄▄▄█▄█  █▄▄█▄▄▄█  █▄█▄▄▄▄▄▄▄█▄▄█ █▄▄█▄▄▄▄▄▄██▄▄█ █▄▄█▄▄▄▄▄▄▄█▄▄▄█▄▄▄▄▄▄▄█▄▄▄█ █▄▄▄█   █▄▄▄█  
```
👌😛😛

### CODE REVIEW

**Comment by Stektpotet** [comment]
Also did some touch-ups in the readme

**Comment by Stektpotet** [comment]
I made a go-solution for day01 too... I didn't intend for that to be in the same pull request, but I guess that's how pull requests works?  ¯\\_(ツ)_/¯

**Comment by Stektpotet** [comment]
Added golang Day02 solution 🧐

**Comment by Stektpotet** [comment]
Added golang Day03 solution 🧠👨‍💻

File: README.md [hunk 1]
```diff
@@ -9,9 +9,9 @@ Anyone is encouraged to contribute, and as a contributor you may do whatever you
 A shout-out to the developers over at Maritime Optima - https://github.com/orgs/MaritimeOptima/people - volunteering to support this project :pray:
 
 Also a big thanks to the rest of our contributors :tada:
-- @Celebrian
-- @Avokadoen
-- @Stekpotet
+- [@Celebrian](https://github.com/Celebrian)
+- [@Avokadoen](https://github.com/Avokadoen)
+- [@Stektpotet](https://github.com/Stektpotet)
```
Comment (Arxcis) [inline README.md:14]: Yes! Bravo :clap:

File: days/day-03/solutions/one-liner.py [hunk 2]
```diff
@@ -0,0 +1 @@
+import fileinput; from functools import reduce; from operator import mul; [(print(sums[1]), print(reduce(mul, sums, 1))) for sums in [[sum([(1 - (y % d)) * (l[((y // d) * r) % len(l)] == '#') for y, l in enumerate(lines)]) for lines in [[line[:-1] for line in fileinput.input()]] for r, d in [(1, 1), (3, 1), (5, 1), (7, 1), (1, 2)]]]]
```
Comment (Arxcis) [inline days/day-03/solutions/one-liner.py:1]: Absolutely the worst thing :smile: Approved!

**Review by Arxcis — APPROVED** [review]
Wow @Stektpotet you have outdone yourself this time :smile: 4 solutions in one PR ! :clap: :tada: Thank you
Your one-liners may be horrible, but they are indeed fast! Checkout `day01`. Faster than C++ even?
```
--- Day 1: Report Repair ---
cat INPUT | go run   day01.go         143ms      ✅
cat INPUT | python3  one-liner.py     284ms      ✅
cat INPUT | g++      day01.cpp        410ms      ✅
cat INPUT | polyc    day01.sml        418ms      ✅
cat INPUT | python3  day01.py         494ms      ✅
cat INPUT | rustc    day01.rs         749ms      ✅
cat INPUT | ruby     day01.rb         1255ms     ✅


--- Day 2: Password Philosophy ---
cat INPUT | gcc      day02.c          1ms        ✅
cat INPUT | g++      day02.cpp        3ms        ✅
cat INPUT | python3  day02.py         10ms       ✅
cat INPUT | go run   day02.go         139ms      ✅
cat INPUT | polyc    day02.sml        405ms      ✅


--- Day 3: Counting trees ---
cat INPUT | gcc      day03.c          1ms        ✅
cat INPUT | python3  day03.py         9ms        ✅
cat INPUT | python3  one-liner.py     11ms       ✅
cat INPUT | python3  python_klyve.py  12ms       ✅
cat INPUT | go run   day03.go         136ms      ✅
cat INPUT | polyc    day03.sml        404ms      ✅
```
Or maybe the others are slow. I don't know.

**Pull request merged**